lib/jekyll.rb

```diff
@@ -32,6 +32,9 @@ def require_all(path)
 require "colorator"
 
 SafeYAML::OPTIONS[:suppress_warnings] = true
+ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
```

Comment:
@mattr- it'll have basic-cross platform support, as much as Jekyll has, I just haven't had time to come back to this as we've bigger problems and this can wait.